src/_utils/radio/__snapshots__/index.unit.tsx.snap

```diff
@@ -0,0 +1,112 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Radio Should render a check icon if isChecked is set to true 1`] = `
+<svg
```

Comment:
Is it expected that there is no input radio in the snapshot?

Reply:
Oh wait `Radio`  is not an `input type="radio` I think this is confusing.

Reply:
Yeah, it's just the Radio icon lookalike ^^
Same thing for Checkbox, it just handles the different status.